### DISCUSSION TITLE
What is profiles for?

### DISCUSSION THREAD

**Discussion by GronbergInfo** [floccusaddon/floccus#1680] [Q&A] [answered]
I just recently started to use floccus and sync bookmarks to a private github repo.
Using it on 2 computers so far, But i have not figured out what the profiles is for. 
On my 2 computers i named the profiles differently, pointing to the same github-file.
Are the profiles only for "local usage" if i sync to several different places?

**Comment by marcelklehr** [comment, marked as answer]
> Are the profiles only for "local usage" if i sync to several different places?

Yes, that's exactly it. You could have one profile syncing bookmarks and one syncing tabs, for example, or one profile syncing your work bookmarks and one profile syncing your private ones.
